### ISSUE TITLE
pose功能实现

### ISSUE THREAD

**Issue by lpj0822** [MiniBullLab/easy_ai#21]
- [ ] 1. 完成alphapose训练，测试和inference，以及下位机测试和inference。
- [x] 2. 调研更加简单、轻量的pose架构。

**Comment by MiniBullLab** [comment]
基于Mask-RCNN的人体姿态检测已经上传到百度网盘上，路径为内部文档/学习资料/基础调研/人体关键点检测。

**Comment by lpj0822** [comment]
KeyPointRCNN网络中rpn开发

**Comment by lpj0822** [comment]
KeyPointRCNN网络中ROIBoxHead开发

**Comment by lpj0822** [comment]
rpn loss与roiBox loss开发完成

**Comment by vitahlin** [comment]
需要集成到EasyAI

**Comment by foww-0001** [comment]
pose网络实现：https://github.com/YuliangXiu/MobilePose-pytorch
@lpj0822

**Comment by foww-0001** [comment]
mobilePose有一层pixel shuffle层不支持。

**Comment by foww-0001** [comment]
human-pose-estimation的代码可以进行多人pose检测的代码已经上传到百度网盘，使用我们自己的workspace环境，运行：
```
python pose_detect_image.py
```
如有没有图片路径，在该文件最后给图片路径：
```
image_folder = "./imgs"
```
@lpj0822

**Comment by lpj0822** [comment]
在增加mobile_pose、resnet_pose

**Comment by lpj0822** [comment]
增加pose dataloader

**Comment by lpj0822** [comment]
添加Pose Accuracy计算

**Comment by foww-0001** [comment]
yolov3 基于COCO的person训练出来的模型放到网盘/深度学习模型文件/pytorch/yolov3_person下面，附带config文件和一个测试视频。
@lpj0822

**Comment by lpj0822** [comment]
pose Augment添加

**Comment by foww-0001** [comment]
pose 预训练模型已经传到百度网盘/深度学习模型文件/pytorch/resnet_pose.pt。

**Comment by foww-0001** [comment]
pose的COCO数据中的一百张已经上传到百度网盘/数据集/公开数据/人体姿态数据/COCO上。
@lpj0822

**Comment by lpj0822** [comment]
pose infer测试没有问题

**Comment by lpj0822** [comment]
python3 inference_task -t pose2d -i 测试数据路径 -m ResnetPose -w 模型权重路径

**Comment by lpj0822** [comment]
pose可以训练了
